package.json

```diff
@@ -5,8 +5,8 @@
   "main": "index.js",
   "scripts": {
     "prepublish": "node ./apidocs/describe-builtin-models.js",
-    "pretest": "jshint .",
-    "test": "node ./test.e2e/test-server.js node node_modules/karma/bin/karma start --single-run --browsers PhantomJS"
+    "test": "node ./test.e2e/test-server.js node node_modules/karma/bin/karma start --single-run  --browsers PhantomJS",
+    "posttest": "npm run prepublish && eslint ."
```

Comment:
Another possiblity is to add a "generate-local-file" script and have posttest and prepublish call `npm run generate-local-file`.